services/auto-idler/idle-services.sh

```diff
@@ -68,7 +70,10 @@ echo "$DEVELOPMENT_ENVIRONMENTS" | jq -c '.data.developmentEnvironments[] | sele
             }
           }'| jq ".hits.total")
 
-          if [ ! $? -eq 0 ]; then
+          if [ "$ENVIRONMENT_AUTOIDLE" == "0" ]; then
+            echo "$OPENSHIFT_URL - $PROJECT_NAME: $ENVIRONMENT_NAME idling disabled, skipping"
+            continue
+          elif [ ! $? -eq 0 ]; then
```

Comment:
this `elif [ ! $? -eq 0 ]; then` is testing the return code of the previous executed command, which needs to come directly after the `curl` command.

Let's add the check for environment idling before the hits loading :)

Reply:
Makes sense to me :)